### DISCUSSION TITLE
updateWorkspaceFolders API: trust authors of workspace

### DISCUSSION THREAD

**Discussion by gee4vee** [microsoft/vscode-discussions#1228] [Extension Development QnA] [answered]
We have a flow in our extension where we add a folder to the workspace using the `updateWorkspaceFolders` method. Is there a way to programmatically agree to trust the workspace author? This is making it so we can't continue in the flow because the workspace folder hasn't been created.

**Comment by isidorn** [comment, marked as answer]
No, there is no way to programmatically agree. The point of that feature is that the user has to explicitly agree to trust the folder.
